include/networkit/graph/Graph.hpp

```diff
@@ -782,34 +782,47 @@ class Graph final {
      * Set name of graph to @a name.
      * @param name The name.
      */
-    void setName(std::string name) { this->name = name; }
+    void TLX_DEPRECATED(setName(std::string name)) { this->name = name; }
 
     /*
      * Returns the name of the graph.
      * @return The name of the graph.
      */
-    std::string getName() const { return name; }
+    std::string TLX_DEPRECATED(getName() const) { return name; }
 
     /**
      * Set edge count of the graph to edges.
      * @param edges the edge count of a graph
      */
     void setEdgeCount(Unsafe, count edges) { m = edges; }
 
+    /**
+     * Set upper bound to the edge count of the graph to edges.
+     *
+     * @param newOmega the new upper bound to the edge count of the graph.
+     */
+    void setOmega(Unsafe, edgeid newOmega) noexcept { omega = newOmega; }
```

Comment:
Rename to `setUpperEdgeIdBound`. Nobody should know what `Omega` is and we already have a getter named `upperEdgeIdBound`.